mivs/templates/mivs_admin/game_results.html

```diff
@@ -66,9 +66,15 @@ <h3>{{ game.title }} judging results</h3>
 {% endfor %}
 </tbody>
 </table>
-    <button type="submit" class="btn btn-primary"
-            {% if game.status not in c.FINAL_GAME_STATUSES %} disabled title="You can only send reviews after a game has been accepted, waitlisted, or declined!"{% endif %}>
-      Mark Reviews for Sending to Indie Studio</button>
+    <button
+        type="submit"
+        class="btn btn-primary"
+        {% if game.status not in c.FINAL_GAME_STATUSES and not c.AFTER_ESCHATON %}
+        disabled="disabled"
+        title="You can only send reviews after a game has been accepted, waitlisted, or declined!"
+        {% endif %}>
+      Mark Reviews for Sending to Indie Studio
+    </button>
```

Comment:
This is mostly whitespace changes. The actual change in logic is _from_ this:
```
{% if game.status not in c.FINAL_GAME_STATUSES %}
```
_to_ this:
```
{% if game.status not in c.FINAL_GAME_STATUSES and not c.AFTER_ESCHATON %}
```